src/image/TanWcs.cc

```diff
@@ -324,8 +324,8 @@ geom::Point2D TanWcs::undistortPixel(geom::Point2D const & pix) const {
 
     // Polynomial orders for U = f(u,v) and V = g(u,v)
     // Note these may be different in general, but not usually in practise.
-    size_t const fOrder = _sipA.rows();
-    size_t const gOrder = _sipB.rows();
+    int const fOrder = _sipA.rows();
+    int const gOrder = _sipB.rows();
```

Comment:
I added a comment.  I couldn't find a version using a cast in the loop that didn't disrupt the readability of the loop more than I was comfortable with.